rulings/cards-rulings.yaml

```diff
@@ -3748,6 +3748,7 @@
   - When meeting a requirement, the controller can choose how he meets it (eg. he can play {Anarch Secession} as a Prince of any city, even a made-up one). [PIB 20150306] [PIB 20150307]
   - He can use a card requiring a sect or titled to be played (eg. use {Abbot} intercept bonus as a Sabbat vampire, perform {Archon} action to burn it as a Prince or Justicar). [PIB 20150306]
   - "He cannot use a card as a vampire of a sect or with a title if the card does not require the sect or title to be played. (eg. he cannot benefit from {Powerbase: Madrid} as a Sabbat vampire) [PIB 20150306]"
+  - If Vlad gains a title via a title card ({Praxis Seizure}, {Fee Stake}, {Crusade}, etc), as long as Vlad is not the correct sect, the title is dormant. He counts as untitled if he has a dormant title. He cannot "use" his title to become titled. [ANK 20190211]
```

Comment:
When using parentheses, you need to put a real/full card name. Otherwise it won't resolve/parse to a real card on online tools (VDB, krcg bot). Either drop the parentheses or use a real card in them.